### ISSUE TITLE
Example in README.md is out of date

### ISSUE THREAD

**Issue by nyh** [scylladb/scylla#3466]
In README.md we have an example of running Scylla in a "tmp" directory:
```
./build/release/scylla --datadir tmp --commitlog-directory tmp --smp 1
```
This example no longer works - it has become necessary, unfortunately, to also add a `--hints-directory tmp` option to this list.

The easy fix is to add this option to README.md.

A harder but perhaps more satisfying fix could be to add a new option, "--home", where all the other directories default to a subdirectories of it (instead of the hard-coded "/var/lib/scylla" from db/config.hh).
This will allow this example - and actual users - to just run "scylla --home tmp" and not worry about the individual subdirectories if they don'y want to (they could still override those, if they want).

**Comment by nyh** [comment]
@duarten your comment is partially true - indeed the code has been fixed to not check the hints-directory on boot (and instead, with a wink, check the data directory). But, if hints will be enabled by default in a future release (they are in Cassandra, I believe!), the problem will resurface.

I already posted a patch for this to make it easy to set all the directories in one "--home" option in addition to the ability to set each and every one of them separately.
